pkgs/servers/minio/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, buildGoModule, fetchFromGitHub, nixosTests }:
+{ lib, buildGo116Module, fetchFromGitHub, nixosTests }:
 
-buildGoModule rec {
+buildGo116Module rec {
```

Comment:
I think it is cleaner to use inherit here.

Reply:
How do you mean that?

Reply:
To do it like https://github.com/NixOS/nixpkgs/blob/faf4d485543e9ff49ace1e3b0d17fcda3cb1a9aa/pkgs/top-level/all-packages.nix#L28312

Reply:
Ok I adapted it accordingly